image/tools/entrypoint.sh

```diff
@@ -37,17 +37,19 @@ source "$DIR/lib/component/$component.sh"
 DATESTAMP=$(date '+%Y/%m/%d')
 DEST=/tmp/intly
 ARCHIVES_DEST=$DEST/archives
-mkdir -p $DEST $ARCHIVES_DEST
-export HOME=$DEST
+mkdir -p $DEST
 
 component_dump_data $DEST
 echo '==> Component data dump completed'
+
+export HOME=$DEST
 if [[ "$encryption_engine" ]]; then
+    mkdir -p $ARCHIVES_DEST
```

Comment:
If there is no "encryption_engine", empty "archives" folder will cause trouble with S3 upload.